clang_tools/util.py

```diff
@@ -47,20 +47,26 @@ def download_file(url: str, file_name: str) -> Optional[str]:
 
     if response.status != 200:
         return None
+    assert response.length is not None
```

Comment:
I'm sure how to handle it, it's just a little unclear: if assert result is False, it will stop and throws an error... Is this what you expect?

```python
>>> assert False
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
```

Reply:
This assert statement is strictly for type checking. It will always evaluate to be true because I already check if the response's return code is valid before this assertion.